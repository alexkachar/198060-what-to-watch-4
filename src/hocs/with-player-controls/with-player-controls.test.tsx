import * as React from 'react';
import * as renderer from 'react-test-renderer';
import withPlayerControls from './with-player-controls';

const MockComponent = () => <div />;
const WrappedComponent = withPlayerControls(MockComponent);

it(`withPlayerControls renders correctly`, () => {
  const tree = renderer
    .create(
        <WrappedComponent />, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
