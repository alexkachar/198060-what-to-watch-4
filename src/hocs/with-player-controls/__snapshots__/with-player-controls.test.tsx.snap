// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withPlayerControls renders correctly 1`] = `<div />`;
